opsagent/README.md

```diff
@@ -0,0 +1,40 @@
+# Ops Agent
+
+With [Dataproc 2.2 image version](https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-release-2.2), we recommend installing [Google Cloud Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent) to obtain system metrics. 
+
+This initialization action will install Ops Agent on a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster and provide similar metrics as the [`--metric-sources=monitoring-agent-defaults`](https://cloud.google.com/dataproc/docs/guides/dataproc-metrics#monitoring_agent_metrics) setting which was supported until Dataproc 2.1.
```

Comment:
Minor nit: whenever used as a subject in a sentence, our docs refer to the agent as "the Ops Agent"… Up to you if you want to be consistent.

Reply:
Updated